training/scip/intro-linux-aalto-computing.rst

```diff
@@ -103,8 +102,26 @@ Preparation
 
 **Software installation:**
 
-* It could be useful to install a SSH client, but we have a backup
-  plan if you don't. `Here some options for installing BASH on your computer <https://coderefinery.github.io/installation/bash/>`__.
+* You will need the BASH shell
+
+  * Linux: Open the Terminal application and type: ``bash``
+  * MacOS: Open the Terminal application and type: ``bash``
```

Comment:
Hm, is zsh the default on mac these days?  and on Aalot zsh could be the default, too.

In practice, I think what we talk about should be the same on zsh and bash.  Does anyone know of a case that we need to make a distinction? (for the shell intro part, not the Triton part)